kie-wb-webapp-test/pom.xml

```diff
@@ -0,0 +1,164 @@
+            <executions>
+                <execution>
+                    <id>copy-standalone-xml</id>
+                    <phase>generate-test-sources</phase>
+                    <goals>
+                        <goal>copy-resources</goal>
+                    </goals>
+                    <configuration>
+                        <outputDirectory>${jboss.home}/standalone/configuration</outputDirectory>
+                        <resources>
+                            <resource>
+                                <directory>../kie-wb/kie-wb-webapp/src/main/resources</directory>
+                            </resource>
+                        </resources>
+                         <jvm-args>-XX:MaxPermSize=2048m -XX:PermSize=2048m</jvm-args>
+                         <systemProperties>
+                            <JAVA_OPTS>-Xmx3000m -Xms1000m -XX:PermSize=2000m -XX:MaxPermSize=4000m</JAVA_OPTS>
+                          </systemProperties>
+                    </configuration>
+                </execution>
+            </executions>
+        </plugin>
+        <!-- WildFly plugin to deploy war -->
+        <plugin>
+            <groupId>org.wildfly.plugins</groupId>
+            <artifactId>wildfly-maven-plugin</artifactId>
+            <version>${version.wildfly.maven.plugin}</version>
+            <configuration>
+                <jbossHome>${jboss.home}</jbossHome>
+              <javaOpts>
+                <javaOpt>-Xms2048m</javaOpt>
+                <javaOpt>-Xmx4096m</javaOpt>
+                <javaOpt>-XX:MaxPermSize=2048m</javaOpt>
+              </javaOpts>
+
+                <serverConfig>standalone-full.xml</serverConfig>
+                <targetDir>../</targetDir>
+                <filename>/kie-wb/kie-wb-distribution-wars/target/kie-wb-6.3.0-SNAPSHOT-wildfly8.war</filename>
+            </configuration>
+            <executions>
+                <execution>
+                    <id>pre-integration-phase</id>
+                    <phase>pre-integration-test</phase>
+                    <goals>
+                        <goal>start</goal>
+                        <goal>deploy</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-surefire-plugin</artifactId>
+      <configuration>
+        <excludes>
+          <exclude>**/*Test.java</exclude>
+        </excludes>
+      </configuration>
+      <executions>
+        <execution>
+          <id>integration-test</id>
+          <goals>
+            <goal>test</goal>
+          </goals>
+          <phase>integration-test</phase>
+          <configuration>
+            <excludes>
+              <exclude>none</exclude>
+            </excludes>
+            <includes>
+              <include>**/*Test.java</include>
+            </includes>
+          </configuration>
+        </execution>
+      </executions>
+    </plugin>
```

Comment:
We should generally use `maven-failsafe-plugin` for running integration tests. It is bound to `integration-test` phase by default.